recipes/cutensor/meta.yaml

```diff
@@ -0,0 +1,105 @@
+#  sha256: 1148373079d63cb4517259d65353a2f9e7f0d109bb05f219019caf80b75b1122  # [ppc64le and cuda_compiler_version == "10.2"]
+#  sha256: 5f0ce5d2d0f94c7d096bf6317d5425e66381ec9aa41942904570f982a03a4269  # [ppc64le and cuda_compiler_version == "11.0"]
+
+#  sha256: 5efce496c330cd9f6df38b93e605127977f0221ca023a4c864bb9e2bc48dd595  # [win64 and cuda_compiler_version == "10.1"]
+#  sha256: 23830ae8da7eaaaef63b572ff19a47f3a86ca7124b8ce2c85bd76286cbf58014  # [win64 and cuda_compiler_version == "10.2"]
+#  sha256: a2ff02ca1003f6e96e63335e6296637173d02cf9d44892225815679dc9c13ee7  # [win64 and cuda_compiler_version == "11.0"]
+
+build:
+  number: 0
+  skip: True  # [win or cuda_compiler_version in ("None", "9.2", "10.0")]
+  script:
+    - mkdir -p $PREFIX/include                                            # [unix]
+    - cp -r include/* $PREFIX/include/                                    # [unix]
+    - mkdir -p $PREFIX/lib                                                # [unix]
+    - mv lib/libcutensor.so* $PREFIX/lib/                                 # [unix]
+
+    # FIXME: cutensor.lib is missing in Julia's packaging
+    #- mkdir %LIBRARY_INC%                                                 # [win64]
+    #- copy %SRC_DIR%\\include\\cutensor.h %LIBRARY_INC%\\                 # [win64]
+    #- mkdir %LIBRARY_INC\\cutensor%                                       # [win64]
+    #- copy %SRC_DIR%\\include\\cutensor\\types.h %LIBRARY_INC%\\cutensor  # [win64]
+    #- mkdir %LIBRARY_BIN%                                                 # [win64]
+    #- copy %SRC_DIR%\\bin\\cutensor.dll %LIBRARY_BIN%\\                   # [win64]
+  ignore_run_exports_from:
+    - {{ compiler('c') }}    # [linux]
+  run_exports:
+    - {{ pin_subpackage('cutensor') }}
+  missing_dso_whitelist:
+    - '*/libgomp.*'  # [linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cuda') }}
+    - sysroot_linux-64 2.17  # [linux64]
+    - libgomp  # [linux]
+  run:
+    # Only GCC_3.0 or older symbols present
+    - libgcc-ng >=3.0     # [linux]
+    # Only GLIBCXX_3.4 or older symbols present
+    - libstdcxx-ng >=3.4  # [linux]
+  run_constrained:
+    # Only GLIBC_2.17 or older symbols present
+    - __glibc >=2.17      # [linux]
+
+test:
+  commands:
+    - test -f $PREFIX/include/cutensor.h                    # [linux]
+    - test -f $PREFIX/include/cutensor/types.h              # [linux]
+    - test -f $PREFIX/lib/libcutensor.so                    # [linux]
+
+    #- if not exist %LIBRARY_INC%\\cutensor.h exit 1         # [win]
+    #- if not exist %LIBRARY_INC%\\cutensor\\types.h exit 1  # [win]
+    #- if not exist %LIBRARY_BIN%\\cutensor.dll exit 1       # [win]
+
+about:
+  home: https://developer.nvidia.com/cutensor
+  license: NVIDIA Software License Agreement for the HPC Software Development Kit
+  license_url: https://docs.nvidia.com/hpc-sdk/eula/index.html
```

Comment:
I don't think this is the right title or link to the cuTensor license, trying to dig up the correct info internally.

Reply:
Yeah I linked to the HPC SDK only because I saw cuTENSOR as part of it and I couldn't find any link to cuTENSOR's SLA/EULA: https://docs.nvidia.com/hpc-sdk/index.html